code/game/objects/structures/showcase.dm

```diff
@@ -47,7 +47,7 @@
 
 /obj/structure/showcase/machinery/oldpod/used
 	name = "opened cyrogenic pod"
-	desc = "Cyrogenic pod that has recently discharged its occupand. The pod appears non-functional."
+	desc = "Cyrogenic pod that has recently discharged its occupant. The pod appears non-functional."
```

Comment:
"Cyrogenic pod that has recently discharged its occupant. The pod appears non-functional." <- The second part of the sentence doesn't really mesh well imo. Also adding an A at the start of the first sentence might be good too.